src/persistence/serialize.cpp

```diff
@@ -62,27 +62,27 @@ float dataToFloat(QByteArray data)
 }
 
 // Converts a string into PNet string data
-QByteArray stringToData(QString str)
+QByteArray stringToData(const QString& str)
 {
     QByteArray data(4,0);
     // Write the size in a Uint of variable lenght (8-32 bits)
     int i=0;
-    uint num1 = (uint)str.toUtf8().size();
+    int num1 = str.toUtf8().size();
     while (num1 >= 0x80)
```

Comment:
@sudden6 Cast to char is not possible because of this line (biggest positive value is 0x7f == 127!). Would need to be uchar, ergo, nothing won here. However no cast at all is valid :smile:.